akka-actor/src/main/scala/akka/serialization/Serialization.scala

```diff
@@ -397,9 +405,12 @@ class Serialization(val system: ExtendedActorSystem) extends Extension {
 
     system.dynamicAccess.createInstanceFor[Serializer](fqn, List(classOf[ExtendedActorSystem] -> system)).recoverWith {
       case _: NoSuchMethodException =>
-        system.dynamicAccess.createInstanceFor[Serializer](fqn, Nil)
-      // FIXME only needed on 2.13.0-M5 due to https://github.com/scala/bug/issues/11242
-      case t => Failure(t)
+        system.dynamicAccess.createInstanceFor[Serializer](fqn, Nil).recoverWith {
+          case _: NoSuchMethodException =>
+            system.dynamicAccess.createInstanceFor[Serializer](
+              fqn,
+              List(classOf[ExtendedActorSystem] -> system, classOf[String] -> bindingName))
```

Comment:
maybe don't try this if it is `""` so the exception is trying the ctr they should have if loading via `serializerOf`

Reply:
good point, fixed in 32fc543c3e